shimmingtoolbox/cli/create_coil_profiles.py

```diff
@@ -291,6 +296,147 @@ def create_coil_profiles_cli(fname_json, path_relative, autoscale, unwrapper, th
     logger.info(f"Filename of the coil config file is: {fname_coil_config}")
 
 
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+)
+@click.option('-i', '--input', 'fname_txt', type=click.Path(exists=True), required=True,
+              help="Input filename of wires' geometry text file. "
+                   "See the `tutorial <https://shimming-toolbox.org/en/latest/user_section/tutorials.html>`_ for more "
+                   "details.")
+@click.option('--fmap', 'fname_fmap', required=True, type=click.Path(exists=True),
+              help="Static \u0394B\u2080 fieldmap on which to calculate coil profiles. Only FOV and affine are used.")
+@click.option('--offset', 'offset', required=False, type=(float, float, float), default=(0,0,0),
+              help="XYZ offset: The difference between the coil’s isocenter position and the field map's isocenter position (in mm). \n Defaulted to (0,0,0).")
+@click.option('--flip', 'dims_to_flip', required=False, type=(float, float, float), default=(1,1,1),
+              help="Dimensions (XYZ order) to flip in the wires' geometry (1 for no flip, -1 for flip). Defaulted to 1 1 1.")
+@click.option('--software', type=click.Choice(['AutoCAD']), default='AutoCAD',
```

Comment:
I suggest keeping the options in lowercase because it becomes a struggle on the command line knowing which letter to uppercase